rotkehlchen/chain/ethereum/node_inquirer.py

```diff
@@ -251,6 +232,41 @@ def _ens_lookup(
             log.error(f'Error deserializing address {address}')
             return None
 
+    def get_ens_resolver_addr(
+            self,
+            name: str,
+    ) -> tuple[Optional[ChecksumEvmAddress], Optional[str]]:
+        """Get the ENS resolver for the given name. Also returns the normalized name.
+
+        May raise:
+        - RemoteError if Etherscan is used and there is a problem querying it or
+        parsing its response
+        - InputError if the given name is not a valid ENS name
+        """
+        try:
+            normal_name = normalize_name(name)
+        except InvalidName as e:
+            raise InputError(str(e)) from e
+
+        resolver_addr = self.contracts.contract(ENS_MAINNET_ADDR).call(
+            self,
+            method_name='resolver',
+            arguments=[normal_name_to_hash(normal_name)],
+        )
+        if is_none_or_zero_address(resolver_addr):
+            return None, None
+
+        try:
+            deserialized_resolver_addr = deserialize_evm_address(resolver_addr)
+        except DeserializationError:
+            log.error(
+                f'Error deserializing address {resolver_addr} while doing'
```

Comment:
```suggestion
                f'Error deserializing address {resolver_addr} while doing '
```

Reply:
Dude why did you not press review 10 secs later? I just pushed.

Reply:
ah wait no I did not push. AHAHAAAAHAHAAHAH